### PR TITLE
limit courses filter search size

### PR DESCRIPTION
**Pull request by Wil0408**

### CODE REVIEW

File: src/routes/courses.js [hunk 1]
```diff
@@ -57,6 +57,8 @@ router.post('/search', async (req, res) => {
     }
 });
 
+const max_search_size = 15000;
```
Comment (jc-hiroto) [inline src/routes/courses.js:60]: Suggest to make this constant a environment variable (i.e. `COURSE_REQUEST_LIMIT`) for better maintenance flexibility 🥰

File: src/routes/courses.js [hunk 2]
```diff
@@ -74,6 +76,11 @@ router.post('/ids', async (req, res) => {
     res.status(200).send({courses: [], total_count: 0});
     return;
   }
+  else if(ids.length >= max_search_size) {
+    console.log('Courses search ids too much');
+    res.status(400).send({courses: [], message: 'Courses search ids too much'});
```
Comment (jc-hiroto) [inline src/routes/courses.js:81]: Maybe change msg to "Request course ids size exceeds limit." would be better.

**Review by jc-hiroto — CHANGES_REQUESTED** [review]

File: src/routes/courses.js (hunk 1 above)
Comment (Wil0408) [reply src/routes/courses.js:60]: Sure! That'll be better.

**Review by jc-hiroto — APPROVED** [review]
Nice work.

**Pull request merged**